### ISSUE TITLE
Add raw input for MGS3

### ISSUE THREAD

**Issue by Durandline** [Lyall/MGSHDFix#43]
The default mouse controls for MGS3 only emulate a right analogue stick.  Adding in raw input would be very helpful.

**Comment by mntorankusu** [comment]
I'd like to point out that they've updated the game to improve the mouse input, but they still don't seem to be bypassing the filtering that the analog stick normally goes through. It feels like a cross gate where horizontal and vertical movement is emphasized, so that if you try to make a circle with the mouse, it ends up being a square. Very small movements are also filtered out as if it were a joystick with a deadzone.

If you can find the function where the mouse input is being filtered, it may be possible to fix this without writing a whole new raw mouse input system.

**Comment by ElTioRata** [comment]
Mouse support has arrived on MGS2 with v2.0.0 patch, so maybe keep the title as "Add mouse raw input" only.

**Comment by baaannnaaannn** [comment]
Yes,i want to create post about it but decide to search and found your post,its really like it
i played in normal and hard but when i decide to try extreame - game needs a more accurate aiming and one mistake - u re dead
and I started to feel it, the mouse acceleration is sometimes fast, sometimes slow, sometimes it switches from a conditionally "slow stick movement" to "fast stick movement"
this does not give precise aiming and probably in this scenario even a gamepad will be better than a mouse
I don't know what efforts are needed for this and what kind of code upgrade, and is it even possible at all. But this is really necessary for high difficulty levels

i think its a copy of https://github.com/Lyall/MGSHDFix/issues/110 (or this one is a copy bcs its much newer)
